packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -152,4 +174,38 @@ export class ApplicationLoadBalancedFargateService extends ApplicationLoadBalanc
       throw new Error(`${name}: Must be a non-negative integer; received ${value}`);
     }
   }
+
+  /**
+   * Throws an error if containerCpu is greater than cpu or is a not positive integer.
+   * @param containerCpu The minimum number of CPU units to reserve for the container.
+   * @param cpu The number of cpu units used by the task. (default: 256)
+   */
```

Comment:
Thanks, but JSDoc is not required for private methods.

```suggestion
```

Reply:
https://github.com/aws/aws-cdk/pull/30920/commits/707a331261163204318ca833786c140fda4f77a1